### DISCUSSION TITLE
Adding Internet Exchange Point (IXP) Names in Network Alarms API Endpoint

### DISCUSSION THREAD

**Discussion by moawnallah** [InternetHealthReport/gsoc#26] [General]
@romain-fontugne It would be beneficial to include Internet Exchange Point (IXP) Names as well in IHR Alarms API, as they are required for Network Delay Alarms API at the Start and End points. This is related to [Return AS names in all IHR alarms API in Global Report V2](https://github.com/orgs/InternetHealthReport/projects/10?pane=issue&itemId=38673458)

**Comment by romain-fontugne** [comment]
yes, I agree, thanks!
